cypress/fixtures/seeds/collectionsFixture.json

```diff
@@ -13,7 +13,6 @@
       "version": "001",
       "createdAt": 1537830419369,
       "granuleId": "^.*$",
-      "provider_path": "granules/fake_granules",
```

Comment:
This is for compatibility with the new API version which deprecated this field on collections